lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2948,16 +2963,18 @@ ParseResult FIRStmtParser::parseRegister(unsigned regIndent) {
   ArrayAttr annotations = getConstants().emptyArrayAttr;
   Value result;
   StringAttr sym = {};
+  // TODO: isConst -> hasConst.
+  bool forceable = !isConst(type);
```

Comment:
Use "does this type recursively contain a constant type" check when that's available,
not a concern until const parsing is added (and should add test for this once it is).
cc https://github.com/llvm/circt/pull/4982#issuecomment-1502077505 .